easybuild/easyconfigs/c/CMake/CMake-3.6.1-use-gnu11.patch

```diff
@@ -0,0 +1,14 @@
+diff -ur cmake-3.6.1.orig/Modules/Compiler/Intel-C.cmake cmake-3.6.1/Modules/Compiler/Intel-C.cmake
```

Comment:
@wpoely86 patch description + author missing, please include any relevant error messages to help Google ;)